app/Controller/ModulesQueryController.php

```diff
@@ -0,0 +1,64 @@
+<?php
+App::uses('AppController', 'Controller');
+class ModulesQueryController extends AppController {

```

Comment:
This should be called ModulesController.

Reply:
Ok. Make sense.